diff --git a/buildRelease.gradle b/buildRelease.gradle
index d6cb0fe..5260a59 100644
--- a/buildRelease.gradle
+++ b/buildRelease.gradle
@@ -3,7 +3,7 @@ tasks.register('buildReleaseTask', Copy) {
   description = 'Building release version ${rootProject.siyuanVersionName}'
 
   def versionName = siyuanVersionName
-  def releaseDirPath = "$rootDir/app/build-release/siyuan-${versionName}-all"
+  def releaseDirPath = "$rootDir/app/build-release/siyuan-all"
 
   delete releaseDirPath
 
@@ -35,5 +35,5 @@ tasks.register('buildReleaseTask', Copy) {
   }
 
   // 依赖于构建任务
-  dependsOn 'assembleXiaomiRelease', 'assembleVoRelease', 'bundleGoogleplayRelease', 'bundleHuaweiRelease', 'assembleOfficialRelease'
+  dependsOn 'assembleOfficialDebug'
 }
