diff --git a/buildRelease.gradle b/buildRelease.gradle
index d6cb0fe..99a8af4 100644
--- a/buildRelease.gradle
+++ b/buildRelease.gradle
@@ -35,5 +35,5 @@ tasks.register('buildReleaseTask', Copy) {
   }
 
   // 依赖于构建任务
-  dependsOn 'assembleXiaomiRelease', 'assembleVoRelease', 'bundleGoogleplayRelease', 'bundleHuaweiRelease', 'assembleOfficialRelease'
+  dependsOn 'assembleOfficialDebug','assembleXiaomiDebug'
 }
