diff --git a/app/build.gradle b/app/build.gradle
index 3a62b3e..1837427 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -4,7 +4,7 @@ plugins {
 
 // 多渠道打包配置已引用的方式传递
 apply from: rootProject.file('flavors.gradle')
-apply from: rootProject.file('signings.gradle')
+//apply from: rootProject.file('signings.templates.gradle')
 apply from: rootProject.file('buildRelease.gradle')
 
 // By appending the version name, build type, and a timestamp to the APK filename,
@@ -23,13 +23,24 @@ android.applicationVariants.configureEach { variant ->
 
         // 根据构建类型调整文件名
         if (buildType != "release") {
-            def formattedDate = new Date().format('yyyyMMddHHmmss')
-            project.setProperty("archivesBaseName", "siyuan-${versionName}-${buildType}-${formattedDate}")
+            newApkName = "app-debug.apk"
+            project.setProperty("archivesBaseName", "app-debug")
         }
     }
 }
 
 android {
+
+    signingConfigs {
+        debug {
+            storeFile = file("../keystore/debug.jks")
+            storePassword System.getenv("KEYSTORE_PASSWORD")
+            keyAlias "debug"
+            keyPassword System.getenv("KEYSTORE_PASSWORD")
+        }
+    }
+
+
     namespace 'org.b3log.siyuan'
     compileSdk 34
 
@@ -53,12 +64,21 @@ android {
             resValue "string", "app_package_name", "org.b3log.siyuan"
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
-        debug {
+/*        debug {
             applicationIdSuffix ".debug"
             minifyEnabled false
             resValue "string", "app_name", "SiYuan-Debug"
             resValue "string", "app_package_name", "org.b3log.siyuan.debug"
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }*/
+        debug {
+            applicationIdSuffix ".debug"
+            minifyEnabled true
+            shrinkResources true
+            resValue "string", "app_name", "SiYuan"
+            resValue "string", "app_package_name", "org.b3log.siyuan"
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
         }
     }
 
