diff --git a/signings.templates.gradle b/signings.templates.gradle
index d3e168c..cc6e627 100644
--- a/signings.templates.gradle
+++ b/signings.templates.gradle
@@ -2,12 +2,10 @@ android {
   // 签名配置
   signingConfigs {
     siyuanConfig {
-      // 如果使用 android studio 生成则无需配置
-      // 如果使用命令行的方式则需要将密钥文件复制到 app 文件夹下 app/siyuan-android.jks
-      storeFile file("siyuan-android.jks") // 密钥文件
-      storePassword "android"      // 密码
-      keyAlias "androiddebugkey"   // Alias
-      keyPassword "android"        // Alias 密码
+      storeFile file("../keystore/debug.jks")
+      storePassword System.getenv("KEYSTORE_PASSWORD")
+      keyAlias "debug"
+      keyPassword System.getenv("KEYSTORE_PASSWORD")
     }
   }
 }
